modules/Classroom/components/ExamList/ExamList.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+
+import { Button, Collapse, Flex, SimpleGrid, Title, Loader } from "@mantine/core";
+import dayjs from "dayjs";
+import { FaChevronRight, FaChevronDown } from "react-icons/fa";
+
+import { useGetExamsQuery } from "@/shared/redux/rtk-apis/exams/exams.api";
+
+import ExamCard from "../ExamCard/ExamCard";
+import { IExamsProps } from "./ExamList.interface";
+import { useStyles } from "./ExamList.styles";
+
+const ExamList: React.FC<IExamsProps> = ({ classroomId }) => {
+  const { classes } = useStyles();
+  const [toggleExamsCollapse, setToggleExamsCollapse] = useState(true);
+
+  const { data: exams, isLoading, error } = useGetExamsQuery({ classroomId });
+
+  const pastExams = exams?.filter((exam) => dayjs(exam.date).isBefore(dayjs()));
+  const futureExams = exams?.filter((exam) => dayjs(exam.date).isAfter(dayjs()));
```

Comment:
I think it makes sense to abstract this out into a method which takes in an array and returns finished and future exams.